jarbas/core/tests/test_reimbursements_command.py

```diff
@@ -89,69 +89,67 @@ def test_serializer(self):
         self.assertEqual(self.command.serialize(input), expected)
 
 
-class TestBulkCreate(TestCommand):
-
-    @patch('jarbas.core.management.commands.reimbursements.Command.bulk_create')
-    def test_bulk_create_by(self, bulk_create):
-        self.command.bulk_create_by(range(0, 10), 3)
-        expected = (
-            call([0, 1, 2]),
-            call([3, 4, 5]),
-            call([6, 7, 8]),
-            call([9])
-        )
-        bulk_create.assert_has_calls(expected)
+class TestCreate(TestCommand):
 
-    @patch.object(Reimbursement.objects, 'bulk_create')
+    @patch.object(Reimbursement.objects, 'update_or_create')
     @patch('jarbas.core.management.commands.reimbursements.Command.print_count')
-    def test_bulk_create(self, print_count, bulk_create):
-        self.command.count = 0
-        self.command.bulk_create([1, 2, 3])
-        bulk_create.assert_called_once_with([1, 2, 3])
-        self.assertEqual(1, print_count.call_count)
+    def test_create_or_update(self, print_count, create):
+        reimbursements = (dict(ahoy=42, document_id=1),)
```

Comment:
If print_count shouldn't be called when reimbursement is not created (without id), you should test this behaviour too. For instance: new test with two reimbursements, only one with id, assert print_count called once.